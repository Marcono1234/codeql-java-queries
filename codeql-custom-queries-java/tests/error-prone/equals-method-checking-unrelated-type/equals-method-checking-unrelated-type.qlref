./error-prone/equals-method-checking-unrelated-type.ql
