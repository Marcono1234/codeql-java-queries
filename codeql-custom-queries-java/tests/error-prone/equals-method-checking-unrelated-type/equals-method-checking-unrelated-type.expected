| Test.java:8:20:8:25 | equals | Checks $@ for unrelated type $@ | Test.java:10:13:10:32 | ...instanceof... | here | Test.java:2:18:2:22 | Other | Other |
| Test.java:8:20:8:25 | equals | Checks $@ for unrelated type $@ | Test.java:11:13:11:42 | ...instanceof... | here | Test.java:3:18:3:29 | OtherGeneric | OtherGeneric |
| Test.java:8:20:8:25 | equals | Checks $@ for unrelated type $@ | Test.java:12:13:12:41 | ... == ... | here | Test.java:2:18:2:22 | Other | Other |
